### DISCUSSION TITLE
Welcome to Foxglove Studio Discussions!

### DISCUSSION THREAD

**Discussion by amacneil** [foxglove/community#53] [Q&A]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we build together  🦍💪.

**Comment by ynx0** [comment]
This is really cool. Thanks for making this open source and as extensible as it is today.

**Reply by jtbandes** [reply]
Welcome! Please feel free to join our Slack community at https://foxglove.dev/community 🙂
